.github/workflows/static_analysis.yaml

```diff
@@ -123,33 +121,60 @@ jobs:
           
           python3 -mvenv venv
           venv/bin/pip3 install -r requirements.txt
+          
+      - name: "Get Current CMake version"
+        id: cmake_version
+        run: |
+          work_dir=$(pwd)
+          cd ..
+          
+          cd cmake_min_version
           venv/bin/python3 cmake_downloader.py --first_minor
           output=$(venv/bin/python3 cmake_min_version.py $work_dir)
-          echo "$output"
+          echo $output
           
           # Retrieve CMake minimal version from the last line of the tool output.
-          current_cmake_version=$(echo ${output: -30} | cut -d'(' -f 2 | head -c -2)
-
-          # Saving cmake minimal version string in GitHub output
-          echo "current_cmake_version=$current_cmake_version" >> $GITHUB_OUTPUT
-
+          current_cmake_version=$(echo ${output} | grep -oP "VERSION \d+\.\d+")
+           
+          regex_pattern="VERSION [0-9]+\.[0-9]+"
+          if ! [[ "$current_cmake_version" =~ $regex_pattern ]];
```

Comment:
Doesn't seem like this is needed...

Reply:
![image](https://github.com/IntelRealSense/librealsense/assets/26552140/4e9de36d-50c4-4c42-995d-cd08216b403f)

I wrote script and found that when _grep_ doesn't found substring by pattern it not perform exit.
So it may make difficult to understand way it failed in a next checks.

Or may test incorrect?